gominio/model/minio.go

```diff
@@ -0,0 +1,70 @@
+package model
+
+import (
+	"bytes"
+	"encoding/xml"
+	"sync"
+	"time"
+)
+
+var ms *MinioServer
```

Comment:
I think it's unproperly to have a package level variable. Because we can init many minio servers in a process.